### ISSUE TITLE
Voice over creates confusion in Terms of Use and Data Privacy section

### ISSUE THREAD

**Issue by Dontuknowwhoiam** [corona-warn-app/cwa-app-ios#688]
## Describe the bug

When you have VO (Voice over) enabled and you navigate to _App information_ ->
_Terms of Service_ you will see when swiping to go through each point or label, it automatically  jumps to the bottom of the scrollview. https://streamable.com/edblm5 
The same thing happens in the Data privacy section. Maybe more of a glitch than a bug. Sorry for that.
## Expected behaviour

When swiping right the next accessiblity element (easier to stick with a11y) should be focused. 
## Steps to reproduce the issue
https://streamable.com/edblm5 

## Technical details

Device: iPhone X
Version: iOS 13.5.1

## Possible Fix

As far as I can remember you can group certain a11y elements and give them a fixed order.

---
Internal Tracking Id: [EXPOSUREAPP-1876](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-1876)

**Comment by dsarkar** [comment]
Currently being analyzed by developers. DS

---
Corona-Warn-App Open Source Team

**Comment by dsarkar** [comment]
This issue can still be reproduced on CWA 1.10.1

**Comment by Ein-Tim** [comment]
I think this is fixed with 1.12 but I don't often use voice over so I can't really tell.

@Dontuknowwhoiam maybe you would like to test and if it's fixed you can close this issue.

**Comment by dsarkar** [comment]
@Dontuknowwhoiam @Ein-Tim 

I retested in iPhone 6s, seems to work now. As @Ein-Tim suggests, @Dontuknowwhoiam please close this issue. Thanks, DS

Corona-Warn-App Open Source Team
